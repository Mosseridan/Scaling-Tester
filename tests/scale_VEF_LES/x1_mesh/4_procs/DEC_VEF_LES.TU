Statistiques de post resolution

Temps total                       5.97867

