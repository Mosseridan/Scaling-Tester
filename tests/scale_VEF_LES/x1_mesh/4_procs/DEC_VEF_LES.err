Petsc initialization succeeded.
Parallel engine initialized : Comm_Group_Noparallel with 1 processors
Executable: /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
Data file name = DEC_VEF_LES
feenableexcept enabled.
Data file : DEC_VEF_LES
Localisation etude: /home/idanmos/projects/Scaling-Tester/tests/scale_VEF_LES/x1_mesh/4_procs
Nom du cas DEC_VEF_LES
 code : /home/idanmos/CEA/no_omp/TrioCFD/TrioCFD_opt
 version : 1.7.6 
Debut de l'execution 
MAIN: Reading and executing data file
Dimension::interpreter : dimension = 3
It treats a calculation with a precision 1e-05
It deals with a calculation using precision format 5
Execution of the interpreter Lire_Tgrid
Reading a mesh which comes from Tgrid
Association of the domain on which the interpreter applies Lire_Tgrid
Reading of the file Tube_coude_463259.msh ...
The total number of elements to read is 463259
3D elements of type Tetrahedron
Reading a comment:
(0

Reading of the dimension of the case:
Dimension 3.

Reading a comment:
(0

The total number of nodes to read is 89710

89710 nodes are read in the area 8


The type of area 9 is 1 (1:fluid, 17:solid)

The total number of faces to read is 950399

Reading a comment:
(0

902637 internal faces are read in the area 10

Reading a comment:
(0

19438 faces are read from the boundary number 11

Reading a comment:
(0

27656 faces are read from the boundary number 12

Reading a comment:
(0

334 faces are read from the boundary number 13

Reading a comment:
(0

334 faces are read from the boundary number 14

Reading a comment:
(0

Reading of a name:
The area 9 is called fluid

Reading of a name:
The area 10 is called interieur

Reading of a name:
The area 11 is called symmetry

Reading of a name:
The area 12 is called wall

Reading of a name:
The area 13 is called outlet

Reading of a name:
The area 14 is called inlet

Elimination of 0 unnecessary nodes in the domain dom
Execution of the interpreter Dilate
Association of the domain on which the interpreter applies Dilate
Expands the domain dom of a factor 0.001
Execution of the interpreter Transformer
Association of the domain on which the interpreter applies Transformer
Reading and interpretation of the function x
Interpretation of the function x Ok
Reading and interpretation of the function y
Interpretation of the function y Ok
Reading and interpretation of the function -z
Interpretation of the function -z Ok
Execution of the interpreter ExtrudeBord
Association of the domain on which the interpreter applies ExtrudeBord
number of 3D nodes :89710
number of 3D faces of inlet :334
number of nodes per faces in 3D :3
boundary maintained symmetry
boundary maintained wall
boundary to eliminate outlet
Decouper : Splitting of a domain
 Domain name to split : box
==============================================
The extreme coordinates of the domain box are:
x is between -0.5334 and -0.4572
y is between -0.1624 and -0.1524
z is between 0 and 0.0381
==============================================
==============================================
The volume cells of the domain box are:
sum(volume cells)= 2.27368e-05
mean(volume cells)= 1.21561e-09
min(volume cells)= 5.47064e-10
max(volume cells)= 4.81391e-09
==============================================
domaine = box
 Creation of a partitioner of type: Partitionneur_metis
Reading parameters of Partitionneur_Metis
Partitionneur_Metis::construire_partition
 Construction of graph connectivity...
 Construction of the som_elem connectivity
 Construction of the elem_elem connectivity
===============
Call for PMETIS
===============
===============
Partitioning quality : edgecut = 504
-> It is roughly the total number of edges (faces) which will be shared by the processors.
-> The lesser this number is, the lesser the total volume of communication between processors.
-> You can increase nb_essais option (default 1) to try to reduce (but at a higher CPU cost) this number.
===============
Correction elem0 on processor 0
Correction of the splitting to put the element 0 on processor 0.
 Exchange of parts 0 and 2
The partitioner has generated 4 parts.
Search of neighboring elements of boundary faces
Construction of lists of elements per subdomain
Generation of 4 parts:
 Construction of part number 0
  Number of nodes    : 1161
  Number of elements : 4676
  Number of joints   : 2
  Joint 0 PeVoisin 2 NbSommets 40 NbFaces 61 NbElemDist 818
  Joint 1 PeVoisin 3 NbSommets 73 NbFaces 121 NbElemDist 1624
               Total:    NbSommets 113 NbFaces 182 NbElemDist 2442
Writing part 0 into the binary file box_0000.Zones
Writing of 1161 nodes.
Writing of 4676 elements.
 Construction of part number 1
  Number of nodes    : 1168
  Number of elements : 4676
  Number of joints   : 1
  Joint 0 PeVoisin 2 NbSommets 104 NbFaces 176 NbElemDist 1890
               Total:    NbSommets 104 NbFaces 176 NbElemDist 1890
Writing part 1 into the binary file box_0001.Zones
Writing of 1168 nodes.
Writing of 4676 elements.
 Construction of part number 2
  Number of nodes    : 1184
  Number of elements : 4676
  Number of joints   : 3
  Joint 0 PeVoisin 0 NbSommets 40 NbFaces 61 NbElemDist 881
  Joint 1 PeVoisin 1 NbSommets 104 NbFaces 176 NbElemDist 2290
  Joint 2 PeVoisin 3 NbSommets 88 NbFaces 146 NbElemDist 1921
               Total:    NbSommets 232 NbFaces 383 NbElemDist 5092
Writing part 2 into the binary file box_0002.Zones
Writing of 1184 nodes.
Writing of 4676 elements.
 Construction of part number 3
  Number of nodes    : 1155
  Number of elements : 4676
  Number of joints   : 2
  Joint 0 PeVoisin 0 NbSommets 73 NbFaces 121 NbElemDist 1479
  Joint 1 PeVoisin 2 NbSommets 88 NbFaces 146 NbElemDist 1871
               Total:    NbSommets 161 NbFaces 267 NbElemDist 3350
Writing part 3 into the binary file box_0003.Zones
Writing of 1155 nodes.
Writing of 4676 elements.
End of the interpreter Decouper
MAIN: End of data file
Arret des processes.
