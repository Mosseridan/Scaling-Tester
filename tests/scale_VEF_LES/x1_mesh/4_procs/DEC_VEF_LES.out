-------------------------------------------------------------------
 
                          TRUST
                      version : 1.7.6 
                          CEA - DEN
 

 0 file(s) modified 


 405 file(s) created 

 ------------------------------------------------------------------
 
  Vous traitez le cas DEC_VEF_LES
 
Reading data file 

13 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
26 MBytes of RAM taken by the calculation.
27 MBytes of RAM taken by the calculation.
 Runtime parameters:
   Objective type: METIS_OBJTYPE_CUT
   Coarsening type: METIS_CTYPE_SHEM
   Initial partitioning type: METIS_IPTYPE_GROW
   Refinement type: METIS_RTYPE_FM
   Perform a 2-hop matching: No
   Number of balancing constraints: 1
   Number of refinement iterations: 10
   Random number seed: -1
   Number of partitions: 4
   Number of cuts: 1
   User-supplied ufactor: 1
   Target partition weights: 
        0=[2.50e-01]   1=[2.50e-01]   2=[2.50e-01]   3=[2.50e-01]
   Allowed maximum load imbalance: 1.001 

     18704      73476      73476 [20] [     1402:   18704 ]
     10391      51118      56850 [20] [     1402:   18704 ]
      5688      30712      45056 [20] [     1402:   18704 ]
      3110      16892      34408 [20] [     1402:   18704 ]
      1691       9218      24710 [20] [     1402:   18704 ]
       908       5870      17614 [20] [     1402:   18704 ]
       482       3792      13260 [20] [     1402:   18704 ]
       256       2336      10472 [20] [     1402:   18704 ]
       137       1364       8524 [20] [     1402:   18704 ]
        72        714       6842 [20] [     1402:   18704 ]
        37        332       5464 [20] [     1402:   18704 ]
        20        152       4284 [20] [     1402:   18704 ]
      9352      36518      36518 [20] [      701:    9352 ]
      5177      25296      28168 [20] [      701:    9352 ]
      2863      15366      22364 [20] [      701:    9352 ]
      1559       8322      17030 [20] [      701:    9352 ]
       848       4492      12190 [20] [      701:    9352 ]
       456       2846       8590 [20] [      701:    9352 ]
       246       1868       6398 [20] [      701:    9352 ]
       131       1144       5034 [20] [      701:    9352 ]
        69        624       4018 [20] [      701:    9352 ]
        35        312       3254 [20] [      701:    9352 ]
        19        142       2584 [20] [      701:    9352 ]
      9352      36544      36544 [20] [      701:    9352 ]
      5198      25198      28236 [20] [      701:    9352 ]
      2860      15196      22374 [20] [      701:    9352 ]
      1556       8264      16932 [20] [      701:    9352 ]
       847       4404      12050 [20] [      701:    9352 ]
       460       2816       8442 [20] [      701:    9352 ]
       241       1844       6272 [20] [      701:    9352 ]
       127       1104       4890 [20] [      701:    9352 ]
        67        612       4002 [20] [      701:    9352 ]
        36        314       3304 [20] [      701:    9352 ]
        18        136       2718 [20] [      701:    9352 ]

Timing Information -------------------------------------------------
 Multilevel: 		   0.038
     Coarsening: 		   0.028
            Matching: 			   0.014
            Contract: 			   0.013
     Initial Partition: 	   0.001
     Uncoarsening: 		   0.008
          Refinement: 			   0.006
          Projection: 			   0.002
     Splitting: 		   0.001
********************************************************************

 gk_mcore statistics
           coresize:       224760         nmops:         2048  cmop:      0
        num_callocs:          356   num_hallocs:            1
       size_callocs:      2562832  size_hallocs:        74816
        cur_callocs:            0   cur_hallocs:            0
        max_callocs:       224472   max_hallocs:        74816
 nbrpool statistics
        nbrpoolsize:            0   nbrpoolcpos:            0
    nbrpoolreallocs:            0

28 MBytes of RAM taken by the calculation.

--------------------------------------------
clock: Total execution: 5.97867 s
